SampleApp/Utilities.swift

```diff
@@ -442,13 +442,9 @@ extension ARSCNView {
 		
 		let maxAngleInDeg = min(coneOpeningAngleInDegrees, 360) / 2
 		let maxAngle = ((maxAngleInDeg / 180) * Float.pi)
-		
-		let points = features.points
-		
-		for i in 0...features.count {
-			
-			let feature = points.advanced(by: Int(i))
-			let featurePos = SCNVector3(feature.pointee)
+				
+        for feature in features.points {
```

Comment:
Same indention problem here. Please check:
https://github.com/charlymr/ARCharts/commit/a09c014e7b5dcd36e3dedc522818e9776fd53cec

Reply:
Alright, let's merge it this way and I'll do an indentation pass in the near future to fix all of the files. Thanks for checking!